css/donate.css

```diff
@@ -0,0 +1,67 @@
+button {
+  text-align: center;
+  color: white;
+  border: none;
+  background-color: rgb(255, 107, 152);
+  align-items: center;
+  padding: 7px 32px;
+  font-size: 25px;
+  font-weight: bold;
+}
+
+.b1 {
+  padding-bottom: 40px;
+  padding-top: 20px;
+}
+
+h1 {
+  color: rgb(255, 107, 152)
+}
+
+h2 {
+  text-align: center;
+  font-size: 40px;
+}
+
+.donate-header{
+  padding-top: 50px;
+  font-size:10px;
+}
+
+.donate-story{
+  background-color:#ECE3F2;
+}
+
+.story-header {
+  padding-top: 50px;
+}
+
+.story-image {
+  padding-right: 50px;
+  padding-bottom: 50px;
+}
+
+.donate-promise {
+  background-color: #E4D7ED;
+}
+
+.promise-header {
+  padding-top: 50px;
+  padding-right: 50px;
+  text-align:right;
```

Comment:
this text should be aligned to the left to match with the paragraph below, and the padding-right is not needed